scipy/fft/_helper.py

```diff
@@ -60,7 +59,11 @@ def next_fast_len(target, kind='C2C'):
     >>> b = fft.fft(a, 131072)
 
     """
-    return _helper.next_fast_len(target, kind)
+    pass
+
+
+next_fast_len = update_wrapper(lru_cache()(_helper.good_size), next_fast_len)
+next_fast_len.__wrapped__ = _helper.good_size
```

Comment:
Can you add a comment about why this is necessary? I assume it's a way to get a reasonable docstring and `args` def while having the call wrap to a C routine.